code/modules/projectiles/ammunition/bullets.dm

```diff
@@ -127,46 +127,55 @@
 	icon_state = "blshell"
 	projectile_type = /obj/item/projectile/bullet/blank
 	starting_materials = list(MAT_IRON = 250)
-	w_type = RECYK_METAL
```

Comment:
Why

Reply:
i have no idea

Reply:
because it's already defined in the parent class*